docs/user/aws/cli.md

```diff
@@ -0,0 +1,20 @@
+# AWS CLI
+
+The AWS CLI is used to configure your AWS credentials on your system for use by the installer. It can also be useful to 
+run other account interrogation and administrative commands.
```

Comment:
Since 4aee1144db (#798), we've been prompting users for these if they don't have credentials in the usual place yet.  Do we still need this section?  I'd expect folks with an existing `~/.aws/credentials` or other standard source would know how to maintain that source already.  I'm fine with a sentence or two to make that explicit, but folks who haven't used any local AWS tooling before should be fine with just the installer and don't need to install the `aws` client.

Reply:
I'm fine either way. @eparis or @crawford, opinions?

Reply:
Let's go ahead and drop this. I'd like to reduce the amount of cognitive overhead.

Reply:
Removed. Pushed.